src/skills-builder/utils/search.js

```diff
@@ -20,45 +20,20 @@ import algoliasearch from 'algoliasearch';
 export const useAlgoliaSearch = () => {
   const config = getConfig();
 
-  const [searchClient, productSearchIndex, jobSearchIndex] = useMemo(
+  const [searchClient] = useMemo(
     () => {
       const client = algoliasearch(
         config.ALGOLIA_APP_ID,
         config.ALGOLIA_SEARCH_API_KEY,
       );
-      const searchNotEmptyClient = {
-        ...client,
-        search(requests) {
-          if (requests.every(({ params }) => !params.query)) {
-            return Promise.resolve({
-              results: requests.map(() => ({
```

Comment:
This is where we used to check for empty queries. This is no longer available in v5 and it is recommended to use `indexUiState` instead ([source](https://www.algolia.com/doc/guides/building-search-ui/going-further/conditional-display/react/#handling-empty-queries))